README.md

```diff
@@ -21,136 +20,157 @@
-|0x072f626a736f6e2f|/bjson/|             |json   | |
-|0x082f75626a736f6e2f| /ubjson/|         |json   | |
-|0x182f6d756c7469636f6465632f | /multicodec/ | | multiformat | 0x40 |
-|0x162f6d756c7469686173682f   | /multihash/  | | multiformat | 0x41 |
-|0x162f6d756c7469616464722f   | /multiaddr/  | | multiformat | 0x42 |
-|0x0a2f70726f746f6275662f |/protobuf/ | Protocol Buffers |protobuf| |
-|0x072f6361706e702f       | /capnp/   | Cap-n-Proto      |protobuf| |
-|0x092f666c61746275662f   |/flatbuf/  | FlatBuffers      |protobuf| |
-|0x052f7461722f         |/tar/      |                 | archive | |
-|0x052f7a69702f         |/zip/      |                 | archive | |
-|0x052f706e672f         | /png/     |                 | archive | |
-|0x052f726c702f         | /rlp/     | recursive length prefix | ethereum |  0x60 |
-## The protocol path
-
-`multicodec` allows us to specify different protocols in a universal namespace, that way being able to recognize, multiplex, and embed them easily. We use the notion of a `path` instead of an `id` because it is meant to be a Unix-friendly URI.
-
-A good path name should be decipherable -- meaning that if some machine or developer -- who has no idea about your protocol -- encounters the path string, they should be able to look it up and resolve how to use it.
-
-An example of a good path name is:
 
-```
-/bittorrent.org/1.0
-```
-
-An example of a _great_ path name is:
-
-```
-/ipfs/Qmaa4Rw81a3a1VEx4LxB7HADUAXvZFhCoRdBzsMZyZmqHD/ipfs.protocol
-/http/w3id.org/ipfs/ipfs-1.1.0.json
+It is worth noting that multicodec-packed works very well in conjunction with [multihash](https://github.com/multiformats/multihash) and [multiaddr](https://github.com/multiformats/multiaddr), as you can prefix those values with a multicodec-packed to tell what they are.
+
+## Multicodec-Packed Protocol Tables
+
+Multicodecuses "protocol tables" to agree upon the mapping from one multicodec code (a single varint). These tables can be application specific, though -- like [with](https://github.com/multiformats/multihash) [other](https://github.com/multiformats/multibase) [multiformats](https://github.com/multiformats/multiaddr) -- we will keep a globally agreed upon table with common protocols and formats.
+
+## Multicodec table
+
+```csv
+codec,              description,              code
+
+miscelaneous
+bin,                raw binary,               0x00
+
+bases encodings
+base1,              unary,                    0x01
+base2,              binary (0 and 1),         0x00
+base8,              octal,                    0x07
+base10,             decimal,                  0x09
+base16,             hexadecimal,              0x
+base32,             rfc4648,                  0x
+base32hex,          rfc4648,                  0x
+base58flickr,       base58 flicker,           0x
+base58btc,          base58 bitcoin,           0x
+base64,             rfc4648,                  0x
+base64url,          rfc4648,                  0x
```

Comment:
IIRC we were removing multibases from multicodec packed. a i misremembering, @diasdavid ?